zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -600,6 +600,11 @@ def do_convert_data(slack_zip_file: str, realm_subdomain: str, output_dir: str,
     slack_data_dir = slack_zip_file.replace('.zip', '')
     if not os.path.exists(slack_data_dir):
         os.makedirs(slack_data_dir)
+
+    # output directory should be empty initially
+    rm_tree(output_dir)
```

Comment:
Sounds dangerous. Either: 1. print a warning sign, 2. exit(1) telling that the directory is not empty.

Reply:
It would be safer to (2) exit(1).

Reply:
It would be safer to (2) exit(1).